libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -465,8 +466,9 @@ function(add_integration_test test_name)
   target_include_directories(${fq_build_target_name} SYSTEM PRIVATE ${LIBC_INCLUDE_DIR})
   target_include_directories(${fq_build_target_name} PRIVATE ${LIBC_SOURCE_DIR})
 
-  _get_hermetic_test_compile_options(compile_options "${INTEGRATION_TEST_COMPILE_OPTIONS}")
-  target_compile_options(${fq_build_target_name} PRIVATE ${compile_options})
+  _get_hermetic_test_compile_options(compile_options "")
+  target_compile_options(${fq_build_target_name} PRIVATE
+                         ${compile_options} ${INTEGRATION_TEST_COMPILE_OPTIONS})
```

Comment:
Why move this?

Reply:
The input argument for `_get_hermetic_test_compile_options` is actually for `FLAGS`, which will be expanded to compile options.  The purpose is that if you want to append extra compile options for your target, you should do that outside of the `_get_..._compile_options` functions.